classes/query/QueryParticipant.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * @file classes/query/QueryParticipant.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class QueryParticipant
+ *
+ * @brief Class for QueryParticipant.
+ */
+
+namespace PKP\query;
+
+use APP\facades\Repo;
+use Eloquence\Behaviours\HasCamelCasing;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Casts\Attribute;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class QueryParticipant extends Model
+{
+    use HasCamelCasing;
+
+    protected $table = 'query_participants';
+    protected $primaryKey = 'query_participant_id';
+    public $timestamps = false;
+
+    protected $fillable = [
+        'queryId', 'userId'
+    ];
+
+    protected function casts(): array
+    {
+        return [
+            'queryId' => 'int',
+            'userId' => 'int'
+        ];
+    }
```

Comment:
In eloquent models, attribute casting should be declared by using $casts property, not a method.
```
protected $casts = [
    'queryId' => 'int',
    'userId' => 'int',
];
```

Reply:
Our laravel version was recently [upgraded](https://github.com/pkp/pkp-lib/pull/10211) to v11 which defines [casts](https://laravel.com/docs/11.x/eloquent-mutators#attribute-casting) via a method.